style.css

```diff
@@ -25,6 +25,15 @@ nav {
     float: right;
     max-height: 100px;
   }
+
+  h4{
+    text-align: left;
+  }
+
+  .inputnewsletter{
```

Comment:
I can suggest to name css classes like that: input-newsletter or inputNewsletter (first one is preferable).
And another thing here: do you maybe plan to have the same style not only for this particular input field? Then it could have more generic name

Reply:
Thanks, I've considered all your comments and changed accordingly. The HW belongs for upcoming one, I didn't put the link  in the HW part will make right now. Sorry for the upcoming HWs, I will leave the comment with appropriate sentence.